src/modules/BootStorage.rb

```diff
@@ -166,6 +171,11 @@ def underlaying_devices(dev)
       res
     end
 
+    # @private
+    def devices_on(dev)
```

Comment:
Missing docu: what it does? returns list of partitions on device? or devices in LVM or ... ?